presto-kinesis/src/main/java/io/prestosql/plugin/kinesis/KinesisConfig.java

```diff
@@ -0,0 +1,325 @@
+    }
+
+    @Config("kinesis.table-description-loc")
+    public KinesisConfig setTableDescriptionLoc(String tableDescriptionLoc)
+    {
+        this.tableDescriptionLoc = tableDescriptionLoc;
+        return this;
+    }
+
+    public boolean isHideInternalColumns()
+    {
+        return hideInternalColumns;
+    }
+
+    @Config("kinesis.hide-internal-columns")
+    public KinesisConfig setHideInternalColumns(boolean hideInternalColumns)
+    {
+        this.hideInternalColumns = hideInternalColumns;
+        return this;
+    }
+
+    @NotNull
+    public String getDefaultSchema()
+    {
+        return defaultSchema;
+    }
+
+    @Config("kinesis.default-schema")
+    public KinesisConfig setDefaultSchema(String defaultSchema)
+    {
+        this.defaultSchema = defaultSchema;
+        return this;
+    }
+
+    public String getAccessKey()
+    {
+        return this.accessKey;
+    }
+
+    @Config("kinesis.access-key")
+    public KinesisConfig setAccessKey(String accessKey)
+    {
+        this.accessKey = accessKey;
+        return this;
+    }
+
+    public String getSecretKey()
+    {
+        return this.secretKey;
+    }
+
+    @Config("kinesis.secret-key")
+    public KinesisConfig setSecretKey(String secretKey)
+    {
+        this.secretKey = secretKey;
+        return this;
+    }
+
+    public String getAwsRegion()
+    {
+        return awsRegion;
+    }
+
+    @Config("kinesis.aws-region")
+    public KinesisConfig setAwsRegion(String awsRegion)
+    {
+        this.awsRegion = awsRegion;
+        return this;
+    }
+
+    public int getBatchSize()
+    {
+        return this.batchSize;
+    }
+
+    @Config("kinesis.batch-size")
+    public KinesisConfig setBatchSize(int batchSize)
+    {
+        this.batchSize = batchSize;
+        return this;
+    }
+
+    public int getMaxBatches()
+    {
+        return this.maxBatches;
+    }
+
+    @Config("kinesis.max-batches")
+    public KinesisConfig setMaxBatches(int maxBatches)
+    {
+        this.maxBatches = maxBatches;
+        return this;
+    }
+
+    public int getFetchAttempts()
+    {
+        return this.fetchAttempts;
+    }
+
+    @Config("kinesis.fetch-attempts")
+    public KinesisConfig setFetchAttempts(int fetchAttempts)
+    {
+        this.fetchAttempts = fetchAttempts;
+        return this;
+    }
+
+    public Duration getSleepTime()
+    {
+        return this.sleepTime;
+    }
+
+    @Config("kinesis.sleep-time")
+    public KinesisConfig setSleepTime(Duration sleepTime)
+    {
+        this.sleepTime = sleepTime;
+        return this;
+    }
+
+    public boolean isLogBatches()
+    {
+        return logKinesisBatches;
+    }
+
+    @Config("kinesis.log-batches")
+    public KinesisConfig setLogBatches(boolean logBatches)
+    {
+        this.logKinesisBatches = logBatches;
+        return this;
+    }
+
+    public boolean isIterFromTimestamp()
+    {
+        return iterFromTimestamp;
+    }
+
+    @Config("kinesis.iter-from-timestamp")
```

Comment:
Don't abbreviate the config option name.

Reply:
expanded

Reply:
This still has the old name. Can you update?